frontend/src/utils/themes.ts

```diff
@@ -50,6 +50,10 @@ const paletteBase = {
         light: lighten("#d0ffb2", 0.1),
         contrastText: "#000",
     },
+    headerButtons: {
+        main: "#fff",
+        contrastText: "#000",
+    },
```

Comment:
You do not need this. The icon buttons can use `color="inherit"` to turn white. The exception is the Home button as it may turn blue because it is inside an anchor. Setting `htmlColor="white"` on the icon (not the icon button) will turn it white.

If you do want to keep this anyway, you should probably name it more appropriately (e.g., white).